src/ProjectCard.svelte

```diff
@@ -12,7 +12,7 @@
         },
         {
             "title" : "App for Himmerland Housing Association",
-            "text" : "A andorid app where an admin user can publish information for the housing association's residents in markdown language and make dropdown items to a residents app. Each page can be edited and deleted as needed.",
+            "text" : "A android app where an admin user can publish information for the housing association's residents in markdown language and make dropdown items to a residents app. Each page can be edited and deleted as needed.",
```

Comment:
```suggestion
            "text" : "An Android app where an admin user can publish information for the housing association's residents in markdown language and make dropdown items to a residents app. Each page can be edited and deleted as needed.",
```

Reply:
Thank you for your contribution Mr. Balslev, it is greatly appreciated. Please look at my suggestion and share your thoughts on this.

Thanks,
Jonas

Reply:
This is a very thoughtful change. I agree!